### PR TITLE
Hot fix to speed up pdf parsing

### PR DESCRIPTION
**Pull request by danich1**
This fix speeds up the PDF processing time by an exponential fold. 

`10.1101/2020.07.31.231381` went from minutes to 11 seconds
`10.1101/566455` went from 20 minutes to 2.9 minutes

The caching pdfminer does is what causes a large lag in time. The software has a bug within the `extract_text` function. Even if you put caching=False the program still uses caching and slows thing down. Instead of waiting for the package to be updated switching functions makes life easier for the time being. Lastly, I added a maxpage parameter, so if 2.9 minutes is too long we can tell pdfminer to only parse X pages to speed things up.

### CODE REVIEW

File: server/word_vectors.py [hunk 1]
```diff
@@ -1,22 +1,32 @@
 import numpy as np
 import pickle
-from io import BytesIO
+from io import BytesIO, StringIO
 from gensim.parsing.preprocessing import remove_stopwords
-from pdfminer.high_level import extract_text
+from pdfminer.high_level import extract_text, extract_text_to_fp
+import time
```
Comment (dongbohu) [inline server/word_vectors.py:6]: This module is not being used, right?

File: server/word_vectors.py [hunk 2]
```diff
@@ -1,22 +1,32 @@
...
-
-    text_to_process = extract_text(BytesIO(content))
-    lines = text_to_process.lower().split("\n")
+    
+    # Have a faux file stream for parsing
+    text_to_process = StringIO()
+    
+    # Use this function to write pdf text to the file stream
+    extract_text_to_fp(
+        BytesIO(content), out,
```
Comment (dongbohu) [inline server/word_vectors.py:24]: Is `out` supposed to be `text_to_process`?

File: server/word_vectors.py (hunk 1 above)
Comment (danich1) [reply server/word_vectors.py:6]: Correct I will be removing it.

File: server/word_vectors.py (hunk 2 above)
Comment (danich1) [reply server/word_vectors.py:24]: Good catch!

**Pull request merged**